dotCMS/src/main/java/com/dotcms/workflow/helper/WorkflowHelper.java

```diff
@@ -117,32 +117,41 @@ public BulkActionView findBulkActions(final User user,
      * @throws DotSecurityException
      */
     private BulkActionView findBulkActionByQuery(final String luceneQuery,
-                                                 final User user) throws DotDataException, DotSecurityException {
-        final String cleanedUpQuery = QueryUtil.removeQueryPrefix(luceneQuery);
-        //We should only be considering Working content.
-        final String query = "{\n"
-                + "    \"query\" : { \n"
-                + "        \"query_string\" : {\n"
-                + "            \"query\" : \"%s\"\n"
-                + "        } \n"
-                + "\n"
-                + "    },\n"
-                + "    \"aggs\" : {\n"
-                + "        \"tag\" : {\n"
-                + "            \"terms\" : {\n"
-                + "                \"field\" : \"wfstep\",\n"
-                + "                \"size\" : 1000  \n"
-                + "            }\n"
-                + "        }\n"
-                + "    },\n"
-                + "\t\"size\":0   \n"
-                + "}";
-        final String preparedQuery = String.format(query,cleanedUpQuery);
-        final ESSearchResults results = this.contentletAPI.esSearch(preparedQuery, false, user, false);
-
-        Logger.debug(getClass(),()-> "luceneQuery: "+ cleanedUpQuery);
-        Logger.debug(getClass(),()-> "esSearch: "+ query);
-        return this.buildBulkActionView(results, user);
+            final User user) throws DotDataException {
+
+        try {
+            final String prepareBulkActionsQuery = LuceneQueryUtils
+                    .prepareBulkActionsQuery(luceneQuery);
+
+            //We should only be considering Working content.
+            final String query = "{\n"
+                    + "    \"query\" : { \n"
+                    + "        \"query_string\" : {\n"
+                    + "            \"query\" : \"%s\"\n"
+                    + "        } \n"
+                    + "\n"
+                    + "    },\n"
+                    + "    \"aggs\" : {\n"
+                    + "        \"tag\" : {\n"
+                    + "            \"terms\" : {\n"
+                    + "                \"field\" : \"wfstep\",\n"
+                    + "                \"size\" : 1000  \n"
+                    + "            }\n"
+                    + "        }\n"
+                    + "    },\n"
+                    + "\t\"size\":0   \n"
+                    + "}";
+            final String preparedQuery = String.format(query, prepareBulkActionsQuery);
+
+            final SearchResponse response = this.contentletAPI
+                    .esSearchRaw(preparedQuery.toLowerCase(), false, user, false);
+
+            Logger.debug(getClass(), () -> "luceneQuery: " + prepareBulkActionsQuery);
+            Logger.debug(getClass(), () -> "esSearch: " + query);
+            return this.buildBulkActionView(response, user);
+        } catch (Exception e) {
+            throw new DotDataException(e);
```

Comment:
Log me

Reply:
The exception bubbles up and it gets logged by the entry point method on the WebResource. So its currently getting logged. 

Adding logs like that adds noise in the log files making the exception appear twice.